<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AskChangesDetailsDlg</class>
 <widget class="QDialog" name="AskChangesDetailsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Apply to:</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>672</width>
     <height>30</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="pushButtonOnlyOne">
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>220</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>Apply only to current node</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAll">
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>220</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>Apply to all childs also</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCancel">
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>220</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
